docker-compose.front-end.yml

```diff
@@ -2,7 +2,11 @@ version: '3.4'
 
 services:
   front-end:
-    image: acctext/frontend:latest
+    build:
```

Comment:
only "dev" docker compose file should contain build, we use this file with command "make run-app", which pulls latest master image from docker hub.
keep this file as it was, and restore "front-end" service with build to "dev"
we will later add profiles to "dev", which were not available previously (https://docs.docker.com/compose/profiles/) to deal with the issue of starting only some needed services.

Reply:
Adding to dev docker-compose leaves no room to develop front-end and start "everything" else.

Reply:
Should we simply add front-end to `docker-compose.yml`?

Reply:
Ok, lets use docker-compose Profiles

Reply:
do not add profiles for now, this is another issue

Reply:
just keep it as it was before. i also noticed that port changed from 80 to 8080, it would probably be best to keep it as it was also

Reply:
if needed, you can start each service separately for now.
`cd api && make run-local` for api
`cd core && make run-gf-service` for gf